concept-registration/pom.xml

```diff
@@ -17,6 +17,8 @@
 	<properties>
 		<java.version>1.8</java.version>
 		<kotlin.version>1.2.71</kotlin.version>
+		<docker.image.prefix>begrepskatalogno</docker.image.prefix>
```

Comment:
Jeg tror vi skulle bruke samme docker repository for alt. kanskje den originalnavn dcatno var litt feil og hele vårt organisasjon måtte kalles fdk?